__tests__/integration.test.js

```diff
@@ -40,6 +40,31 @@ describe('GET: /api', () => {
     });
 });
 
+describe('GET: api/articles/:article_id', () => {
+    test('200: responds with a single article by id', () => {
+        return request(app)
+        .get('/api/articles/3')
+        .expect(200)
+        .then(({ body }) => {
+            expect(body).toHaveProperty("author")
+            expect(body).toHaveProperty("title")
+            expect(body).toHaveProperty("article_id")
+            expect(body).toHaveProperty("body")
+            expect(body).toHaveProperty("topic")
+            expect(body).toHaveProperty("created_at")
+            expect(body).toHaveProperty("votes")
+            expect(body).toHaveProperty("article_img_url")
```

Comment:
This isnt actually asserting that what we get back is what we are expecting. I would recommend using the matcher .toMatchObject() and with expect.any(Number) as an example which gives us more of an insight as to what this value holds.